### PR TITLE
Suba

### PR DESCRIPTION
**Pull request by Suba-1985**
please review stack page

### CODE REVIEW

File: features/step_definitions/HomePageStep.spec.js [hunk 1]
```diff
@@ -2,9 +2,10 @@ const{Given, When, Then} = require('@cucumber/cucumber');
 const{expect} = require('@playwright/test');
 const playwright = require('@playwright/test');
 const {POManager} = require('../../pageObjects/POManager');
+const config = require('../../playwright.config.js');
 
 
-Given('User launches the browser', async function () {
+Given('User launches the browser',{timeout:100*1000}, async function () {
```
Comment (dhvanikam) [inline features/step_definitions/HomePageStep.spec.js:8]: Remove the timeout

File: features/step_definitions/HomePageStep.spec.js [hunk 2]
```diff
@@ -14,9 +15,8 @@ Given('User launches the browser', async function () {
   });
 
   When('User gives the correct DsAlgo portal URL', async function () {
-    await this.page.goto("https://dsportalapp.herokuapp.com/home");
+    await this.page.goto(config.use.baseURL);//replace it with reading from config file
```
Comment (dhvanikam) [inline features/step_definitions/HomePageStep.spec.js:18]: Remove the comments

File: features/step_definitions/HomePageStep.spec.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ Given('User launches the browser', async function () {
     expect(await this.homePage.isSignInLinkVisible()).toBeTruthy();
   });
 
-  When('User clicks on SignIn link on Home Page', async function () {
+  When('User clicks on SignIn link on Home Page',{timeout:100*1000}, async function () {
```
Comment (dhvanikam) [inline features/step_definitions/HomePageStep.spec.js:34]: Remove the timeout

**Review by dhvanikam — CHANGES_REQUESTED** [review]
Check the feedback

File: features/step_definitions/HomePageStep.spec.js (hunk 1 above)
Comment (Suba-1985) [reply features/step_definitions/HomePageStep.spec.js]: resolved the changes

File: features/step_definitions/HomePageStep.spec.js (hunk 2 above)
Comment (Suba-1985) [reply features/step_definitions/HomePageStep.spec.js]: resolved

File: features/step_definitions/HomePageStep.spec.js (hunk 3 above)
Comment (Suba-1985) [reply features/step_definitions/HomePageStep.spec.js]: resolved

File: features/step_definitions/HomePageStep.spec.js (hunk 1 above)
Comment (dhvanikam) [reply features/step_definitions/HomePageStep.spec.js]: Looks good

**Review by dhvanikam — APPROVED** [review]
Changes approved

**Pull request merged**